drivers/i2c/i2c_mchp_xec_v2.c

```diff
@@ -102,6 +103,10 @@ struct i2c_xec_data {
 	uint8_t i2c_status;
 };
 
+#define DRV_CONFIG(dev) ((const struct i2c_xec_config *)(dev)->config)
+#define DRV_DATA(dev) ((struct i2c_xec_data *)(dev)->data)
+#define HAL_INSTANCE(dev) (struct i2c_smb_regs *)(DRV_CONFIG(dev)->base_addr)
```

Comment:
please no more DRV_CONFIG/DRV_DATA, tree is now finally free of all that crap

Reply:
okay, will remove